libs/crypto/include/fetch_bls.hpp

```diff
@@ -27,6 +27,7 @@
 #pragma clang diagnostic push
 #pragma clang diagnostic ignored "-Wconversion"
 #pragma clang diagnostic ignored "-Wpedantic"
+#pragma clang diagnostic ignored "-Wmacro-redefined"
```

Comment:
It's not just clang that moans about this

Reply:
gotcha